robottelo/vm.py

```diff
@@ -333,6 +333,26 @@ def configure_puppet(self, rhel_repo=None):
         # stage can be ignored.
         self.run(u'puppet agent -t 2> /dev/null')
 
+    def execute_foreman_scap_client(self, policy_id=None):
+        """Executes foreman_scap_client on the vm/clients to create security
+        audit report.
+
+        :param policy_id: The Id of the OSCAP policy.
+        :return: None.
+
+        """
+        if policy_id is None:
+            result = self.run(
+                u'awk -F "/" \'/download_path/ {print $4}\' '
+                ' /etc/foreman_scap_client/config.yaml'
```

Comment:
There are two continuous spaces here at the end of first line of awk and at the start of second line.
You can do this instead:
```
u'awk -F "/" \'/download_path/ {print $4}\' '
'/etc/foreman_scap_client/config.yaml'
```